build.rs

```diff
@@ -1165,6 +1166,7 @@ fn main() {
         .blocklist_function("ynl")
         .opaque_type("__mingw_ldbl_type_t")
         .rustified_enum("*")
+        .default_enum_style(bindgen::EnumVariation::Rust { non_exhaustive: false })
```

Comment:
oh, also the linter complains about this line, could you perhaps run `cargo fmt` so it's fixed? :smile: